ParisBNF/abb/BNFabb88.xml

```diff
@@ -36,62 +36,66 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                                        place="margin">ወይሄሎ፡ እስከ፡ ለዓለመ፡ ዓለም፡</add> ዘአልቦ፡ ጥንት፡
-                                        ወኢተፍጻሜት፡<hi rend="rubric"> እምላክነ፡ በመለኮቱ፡ ወአቡነ፡ በኂሩቱ።</hi>
-                                    ዘስብሐቲሁ፡ እምኀቤሁ፡ ወውዳሴሁ፡ እምዚአሁ፡ ዘይክል፡ ኵሎ፡ ወአልቦ፡ ዘይስአኖ፡ ለዓለመ፡ ዓለም፡
-                                    አሜን፨ አጽሕፍ፡ እንከ፡ ንስቲተ፡ አነ፡ ኃጥእ፡ ገብራ፡ <space unit="chars"
-                                        quantity="7"> </space> እምዜና፡ ስደራ፡ ወእ<hi rend="rubric">ምብዙኅ፡
-                                        ገድላቲሃ፡ ወትሩፋቲሃ፡ ለእምነ፡ ቅድስት፡ ወለተ፡ ጴጥ<cb/>ሮስ፡ ሰጻፍ፡ እመ፡ አዕናቊ፡
-                                        አዕላፍ ። ። ።</hi>
-                                </incipit>
+                                <locus from="5r" to="172r"/>
+                                <msItem xml:id="ms_i1.1.1">
+                                    <locus from="5r" to="147rb"/>
+                                    <title type="incomplete" ref="LIT1509GadlaW">Gadla Walatta
+                                        Ṗeṭros</title>
+                                    <incipit xml:lang="gez">
+                                        <locus from="5ra" to="5rb"/>
+                                        <hi rend="rubric">በስመ፡ እግዚአብሔር፡ ዘሃሎ፡ እምቅድመ፡</hi> ዓለም፡ <add
+                                            place="margin">ወይሄሎ፡ እስከ፡ ለዓለመ፡ ዓለም፡</add> ዘአልቦ፡ ጥንት፡
+                                            ወኢተፍጻሜት፡<hi rend="rubric"> እምላክነ፡ በመለኮቱ፡ ወአቡነ፡
+                                            በኂሩቱ።</hi> ዘስብሐቲሁ፡ እምኀቤሁ፡ ወውዳሴሁ፡ እምዚአሁ፡ ዘይክል፡ ኵሎ፡ ወአልቦ፡
+                                        ዘይስአኖ፡ ለዓለመ፡ ዓለም፡ አሜን፨ አጽሕፍ፡ እንከ፡ ንስቲተ፡ አነ፡ ኃጥእ፡ ገብራ፡ <space
+                                            unit="chars" quantity="7"> </space> እምዜና፡ ስደራ፡ ወእ<hi
+                                            rend="rubric">ምብዙኅ፡ ገድላቲሃ፡ ወትሩፋቲሃ፡ ለእምነ፡ ቅድስት፡ ወለተ፡
+                                            ጴጥ<cb/>ሮስ፡ ሰጻፍ፡ እመ፡ አዕናቊ፡ አዕላፍ ። ። ።</hi>
+                                    </incipit>
+                                    <note>According to <bibl><ptr
+                                                target="bm:WendyundKleiner2015WalattaPetros"/><citedRange unit="page">258, n.258</citedRange></bibl>, <cit><quote>CR omits about two manuscript pages of material that is <supplied reason="subaudible">present</supplied>[...] in all the other manuscripts at our disposal. Since the missing material is a folio´s worth, and resumes halfway through an anecdote midsentence, we assume that iether a folio got lost from Abb. 88 or that the scribe of Abb.88 inadvertently skipped one folio while copying.</quote></cit>
+                                        Nafisa Valieva, while cataloguing this manuscript, identified that the omitted passages has been supplied on f.1r, <ref target="@a1"/>. The ommited portion starts on f. 142v with the word ትቤለኒ, while the scribe resumes with ትቤለ<add></add>,  which suggests a homeoteleuton. The scholars further note, <bibl><ptr
```

Comment:
what do you mean under `@a1`?

Reply:
> what do you mean under `@a1`?
this

      <item xml:id="a1">
                                    <locus target="1ra"/>
                                    <desc type="Correction">Addition of the the omitted passages of <title  ref="LIT1509GadlaW"></title> from f.142vb
                                        
                                        <q xml:lang="gez">ወመጽአ፡ ኀቤየ፡ በአምሳለ፡ ብእሲ፡ ብሩህ፡ ወወረደ፡ ሠጢቆ፡ ጠፈረ፡ ቤት፡</q>
                                    </desc>
                                </item>
is it a wrong description?

Reply:
the target needs # not @